wurst/systems/units/UnitEntity.wurst

```diff
@@ -32,8 +32,8 @@ public abstract class UnitEntity
     static function register(int unitTypeId, Function<unit, thistype> factory)
         factories.put(unitTypeId, factory)
 
-    static function findForUnit(unit hostile) returns thistype
-        return instances[hostile.getIndex()]
+    static function findForUnit(unit whichUnit) returns thistype
```

Comment:
Why not just `u`? This comes up enough that a standardized name would be good.

Reply:
`whichUnit` is also standard, that is what they use in common.j and throughout the stdlib